src/token/AirdropPendingTransaction.js

```diff
@@ -0,0 +1,77 @@
+/*-
+ * ‌
+ * Hedera JavaScript SDK
+ * ​
+ * Copyright (C) 2020 - 2024 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import Transaction from "../transaction/Transaction.js";
+
+/**
+ * @typedef {import("../token/PendingAirdropId.js").default} PendingAirdropId
+ */
+export default class AirdropPendingTransaction extends Transaction {
```

Comment:
I think `_validateChecksums` method is missing here.

Reply:
I can't seem to find the logic on why these are used. Some transactions have it and some don't? I only noticed that newer transactions don't use these anymore? Do you have this in Go?

Reply:
Resolved in DMs. I think this is something that isn't enforced in the JS SDK. Newer transaction don't incorporate it. But might have to look into it and start implementing it from now on.